eclipse-collections/src/main/java/org/eclipse/collections/impl/set/mutable/MultiReaderUnifiedSet.java

```diff
@@ -131,6 +131,16 @@ private MultiReaderUnifiedSet(MutableSet<T> newDelegate, ReadWriteLock newLock)
         return new MultiReaderUnifiedSet<>(UnifiedSet.newSetWith(elements));
     }
 
+    public static <T> MultiReaderUnifiedSet<T> of(MutableSet<T> newDelegate)
+    {
+        return new MultiReaderUnifiedSet<>(newDelegate);
+    }
+
+    public static <T> MultiReaderUnifiedSet<T> of(MutableSet<T> newDelegate, ReadWriteLock newLock)
+    {
+        return new MultiReaderUnifiedSet<>(newDelegate, newLock);
+    }
```

Comment:
These are non-copy constructors. We have to remove them, or at least hide their visibility.

Reply:
@motlin good point, actually I added them in the past but don't use them, so I will just remove them.